ugwumadu116/Epic-Mail
# #164456651 Build out get user sent messages

#### What does this PR do?
- Creates a route to get all user sent emails
#### Description of Task to be completed?
- Create a route that uses GET /messages/sent eg `GET /api/v1/messages/sent`
- Create a dummy message data to store message sent by the user
- Create a message service to handle all message functionalities
- Create a message controller to handle route request
- Create a user validator to know the user that is making the request
#### How should this be manually tested?
- Sign in with firstName, lastName, email, and password
   - exmaple `{
	"firstName": "joel",
	"lastName": "ugwumadu2",
	"email": "ugwumadu116@gmail.com",
	"password": "test"
}`
- Login in with the epicMail sent to you with your password 
  - example `{
	"epicMail": "joelugwumadu2@epicmail.com",
	"password": "test"
}`
- Use your login token and set it in Headers and set the key to be x-auth-token while the value will be your token
- Then make a GET request to `/api/v1/messages/sent`
- Your messages will appear in your response
#### What are the relevant pivotal tracker stories?
-	Don’t forget to set your environment variable to `SECRET=secret`
#### What are the relevant pivotal tracker stories?
#164456651
